### PR TITLE
[ADDED]: BlockChain Category

### LINKED ISSUE #1826: [ADD] Blockchain/SmartContracts / Truffle / Hardhat
### Category Name

Blockchain

### Subcategory Name

Smart Contracts , Truffle , Hardhat ,Ethers , Moralis

### Additional Context

In the last few months I have been learning _**blockchain development**_ and I am totally fascinated by this technology , how cool and futuristic this is. I think **Blockchain** as a technology deserves a Category of it's own. I know quite resources on how to develop smart contracts , how to begin using frameworks like truffle and hardhat for blockchain development.

Can you please assign this issue to me so I can add Blockchain as a category as I firmly believe that are quite a lot of people interested in learning this technology but doesn't know where to get the resources from.

### Checklist

- [X] I have checked the existing [issues](https://github.com/rupali-codes/LinksHub/issues?q=is%3Aissue+)
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I have checked that this category does not exist
- [X] I have checked that this subcategory does not exist
- [X] I am willing to work on this issue (optional)
- [X] I am participating in Hacktoberfest 2023

### PR DESCRIPTION
**Pull request by Killer2OP**
## Fixes Issue

Closes #1826 

## Changes proposed
Added Blockchain category along with multiple subcategories 

## Screenshots

image

### CODE REVIEW

**Comment by Killer2OP** [comment]
@Anmol-Baranwal Can you review the PR.

File: database/BlockChain/smart-contracts.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[
+    {
+    "name": "Solidity Documentation",
+    "description": ": Solidity is the programming language used for developing smart contracts on the Ethereum platform. The Solidity documentation provides comprehensive guidance on writing secure and efficient smart contracts.",
+    "url": "https://docs.soliditylang.org/en/v0.8.21/",
+    "category": "BlockChain",
+    "subcategory": "Smart contracts"
```
Comment (Anmol-Baranwal) [inline database/BlockChain/smart-contracts.json:7]: correct this.

File: database/BlockChain/hardhat.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+[
+    {
+    "name": "Hardhat Docs",
+    "description": "It is the official website of hardhat.",
+    "url": "https://hardhat.org/",
+    "category": "BlockChain",
+    "subcategory": "Hardhat"
```
Comment (Anmol-Baranwal) [inline database/BlockChain/hardhat.json:7]: this seems to be wrong. have you checked locally?

**Review by Anmol-Baranwal — CHANGES_REQUESTED** [review]
Check locally to see if the subcategories and resources are showing up.

image

**Comment by Killer2OP** [comment]
@Anmol-Baranwal Review the PR :)

**Review by Anmol-Baranwal — CHANGES_REQUESTED** [review]
the description of smart contracts & ethers is not showing up. check the naming.

image

**Review by Anmol-Baranwal — DISMISSED** [review]
🚀 Looks good to me.

**Review by Anmol-Baranwal — APPROVED** [review]
🚀 Looks good to me.

**Review by CBID2 — DISMISSED** [review]
LGTM

File: components/TopBar/CategoryDescriptions.ts [hunk 3]
```diff
@@ -201,13 +201,23 @@ const categoryDescriptions: CategoryDescriptions = {
   'technical writing tools':
     'This list provides the tools to suit your tech writing needs.',
 
+  //DSA
+  'DSA articles':'This section provides various important articles related to DSA problem solving skills, how to tackle them, how to think about it and many more.',
```
Comment (rupali-codes) [inline components/TopBar/CategoryDescriptions.ts:205]: the description for `DSA articles` has already been added, please remove it or update the existing desc

Comment (Killer2OP) [reply components/TopBar/CategoryDescriptions.ts]: > the description for `DSA articles` has already been added, please remove it or update the existing desc

DONE. @rupali-codes

**Review by Anmol-Baranwal — APPROVED** [review]
🚀 Looks good to me.

**Comment by Killer2OP** [comment]
@rupali-codes merge the PR please :)

**Review by rupali-codes — APPROVED** [review]
Lgtm

**Pull request merged**